sentry-core/Cargo.toml

```
@@ -32,6 +32,7 @@ rand = { version = "0.9.0", optional = true }
 sentry-types = { version = "0.41.0", path = "../sentry-types" }
 serde = { version = "1.0.104", features = ["derive"] }
 serde_json = { version = "1.0.46" }
+url = { version = "2.1.1" }
```

[nit] Why not specify the latest version here? We already appear to be using 2.5.4 in Cargo.lock.

```suggestion
url = { version = "2.5.4" }
```